### ISSUE TITLE
Bethel Theatre name

### ISSUE THREAD

**Issue by emily-johnson** [betheluniversity/tinker#155]
We need to change "Black Box Theatre" to "Bethel Theatre" (black box is apparently the type of theatre we have, not the name) in the list of locations when creating an event. 

If you get us list of events using black box, @kathrynmagnuson and I can go into cascade and update them after the change

Thanks!
Em

**Comment by c-schwarze** [comment]
It looks like there are no events tagged as "Black Box Theatre".

**Comment by emily-johnson** [comment]
Oh, well that makes it easier! So can we go ahead with just changing the name then?

**Comment by c-schwarze** [comment]
Yep! I believe you can update that here: https://cms.bethel.edu/entity/open.act?id=ba1355ea8c586513100ee2a725b9ebea&type=block&
